docs/getting-started/integrating-codacy-with-visual-studio-code.md

```diff
@@ -1,90 +1,91 @@
+After completing these steps, Codacy displays a summary of all problems for the pull request in the main view and a list of found Quality issues in VS Code's Problems tab.
 
 ![Codacy main view](images/codacy-vscode-extension-main-view.png)
 
 The main view displays the following:
 
--   The **Quality status** of the pull request, either up to standards or not up to standards, based on the repository’s [Quality gates](../repositories-configure/adjusting-quality-gates.md).
+-   The **Quality status** of the pull request, either up to standards or not up to standards, based on the [Quality gates](../repositories-configure/adjusting-quality-gates.md) set for the repository.
 
--   Any **Quality issues** introduced or fixed by the pull request. These are the same issues you find in the [Quality Issues tabs](../repositories/pull-requests.md#issues-tabs) in the Codacy app.
+-   Any **Quality issues** introduced or fixed by the pull request. These are the same issues you find in the [Quality Issues tabs](../repositories/pull-requests.md#issues-tabs) in the Codacy app and are also visible in VS Code's Problems tab.
 
-    !!! tip
-        -   The number immediately next to each file name is the total number of issues in that file.
+    The number next to each file name is the total number of Quality issues that the pull request adds to or removes from that file. The number farther to the right is the number of new Medium or Critical Quality issues. If there are any such issues, the file name is also highlighted in yellow (Medium) or red (Critical).
 
-        -   The number farther to the right of each file name is the number of issues with severity Medium or higher in that file. The file is also highlighted in yellow if there are any such issues.
+    ![Important issues](./images/codacy-vscode-extension-important-issues.png)
 
 -   The **diff coverage and coverage variation** introduced by the pull request. These are the same values you find in the [Pull request coverage overview](../repositories-coverage/pull-requests.md#coverage-overview) panel in the Codacy app.
 
-    !!! tip
-        The percentage next to each file name is the coverage variation for that file.
+    The percentage next to each file name is the coverage variation for that file.
 
 -   Sequences of **duplicate code** (clones). These are the same ones you find in the [Quality Duplication tabs](../repositories/pull-requests.md#duplication-tabs) in the Codacy app.
 
 -   Variations in **code complexity**.
 
-## Navigating to found problems
+## Reviewing pull request issues
+
+In the Problems tab, Codacy displays the same Quality issues you find in the main view and lets you navigate to the exact line of code where the issue was found.
 
-To navigate to a found problem:
+!!! note
+    Code coverage, duplicates, and complexity aren't currently shown in the Problems tab.
```

Comment:
Avoid passive voice.

```suggestion
    The Problems tab doesn't list code coverage, duplicates, and complexity.
```

Reply:
Here it's intentional. I'm using the passive to start the sentence with the information of interest (a list of excluded categories). Thoughts?